integr8ly/integreatly-operator
# MGDAPI-6415 move 3scale from dc to deployments

# Issue link
https://issues.redhat.com/browse/MGDAPI-6415

# What
Move 3scale from deployment configs to deployments 
Note: Requires new version of 3scale to pass prow/work

# Verification steps
- Run `make cluster/prepare/local`
- Apply CS with index created from this branch (alternatively create it yourself)
```
cat << EOF | oc create -f -
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: rhoam-operators
  namespace: openshift-marketplace
spec:
  sourceType: grpc
  image: quay.io/mstoklus/managed-api-service-index:1.40.0
EOF
```
- Create RHMI CR:
```
IN_PROW=false USE_CLUSTER_STORAGE=true make deploy/integreatly-rhmi-cr.yml
```
- Wait for RHOAM to fully install
- Update catalog source to 1.41
- Upgrade RHOAM
- Confirm RHOAM got successfully updated and 3scale is on 0.12.1
- Confirm e2e on this PR pass
- Confirm skip preflights checks env is present on the 3scale operator pod and in 3scale operator subscription
